src/pages/Withdrawals/index.tsx

```diff
@@ -0,0 +1,744 @@
+// Import libraries
+import React from 'react';
+import { FormattedMessage, useIntl } from 'react-intl';
+import { HashLink as Anchor } from 'react-router-hash-link';
+import styled from 'styled-components';
+// Components
+import { Alert } from '../../components/Alert';
+import { Heading } from '../../components/Heading';
+import { Link } from '../../components/Link';
+import { PageTemplate } from '../../components/PageTemplate';
+import { Text } from '../../components/Text';
+import { Code } from '../../components/Code';
+import { WithdrawalsTabComparison } from '../../components/WithdrawalsTabComparison';
+
+const ComponentStyles = styled.div`
+  * {
+    scroll-behavior: smooth;
+    scroll-margin-top: 6rem;
+  }
+  section {
+    margin-top: 30px;
+  }
+  a {
+    text-decoration: none;
+  }
+  ul {
+    margin-bottom: 10px;
+  }
+  li {
+    margin-top: 10px;
+    font-weight: 300;
+  }
+  strong {
+    font-weight: 500;
+  }
+  table {
+    margin: 1rem auto;
+    color: #212529;
+
+    * {
+      text-align: start;
+    }
+
+    th,
+    td {
+      padding: 0.75rem;
+      vertical-align: top;
+      border-top: 1px solid #dee2e6;
+    }
+
+    thead th {
+      vertical-align: bottom;
+    }
+
+    tbody + tbody {
+      border-top: 2px solid #dee2e6;
+    }
+  }
+  section.actionable {
+    background-color: white;
+    padding: 2rem;
+    @media only screen and (max-width: ${p => p.theme.screenSizes.medium}) {
+      padding: 1rem;
+    }
+    margin: 2rem 0;
+    border-radius: 4px;
+  }
+`;
+
+const HashCode = styled(Code)`
+  word-break: break-all;
+`;
+
+const SectionTitle = styled(Heading)`
+  margin-top: 30px;
+  border-bottom: 1px solid lightgray;
+  padding-bottom: 10px;
+`;
+
+const SubSectionTitle = styled(SectionTitle)`
+  border-bottom: 0;
+`;
+
+export const Withdrawals = () => {
+  const { formatMessage } = useIntl();
+  return (
+    <PageTemplate
+      title={formatMessage({ defaultMessage: 'Staking withdrawals' })}
+      description={formatMessage({
+        defaultMessage:
+          'Information for validator operators regarding staking withdrawals.',
+      })}
+    >
+      <ComponentStyles>
+        <section>
+          <Text>
+            <FormattedMessage
+              defaultMessage="The Shanghai/Capella upgrade enabled Beacon Chain staking
+              withdrawals, with the inclusion of {eip}. This Ethereum improvement proposal
```

Comment:
For some reason, we have a new line here

![image](https://user-images.githubusercontent.com/468158/216340353-a0c1cb84-34d4-4da2-ac99-1208ce611a34.png)

Reply:
Yeah, the link below needs an `inline` attribute.. I'll add